### ISSUE TITLE
File/Folder comments aren't preserved.

### ISSUE THREAD

**Issue by aptonline** [jhalter/mobius#31]
When adding a comment via the information button and saving they aren't being preserved between views.  Performing a refresh of the files/folder doesn't seem to help.

**Comment by jhalter** [comment]
Hi, at the moment I discard the comment because I don't have anywhere to store it.

OSX stores Files comments in extended file attributes, but I doubt this works the same on Linux or Windows.

```
ls -al@ file-with-comment
-rw-r--r--@ 1 jhalter  staff  1 Jun 10 08:31 file-with-comment
        com.apple.metadata:kMDItemFinderComment 48 
```

As an alternative I'm considering storing the comments and type/creator signature in a separate file as I intend for resource forks:

```
-rw-r--r--   1 jhalter  staff     102 Jun  9 17:44 .info_SimpleText
-rw-r--r--   1 jhalter  staff  120392 Jun  9 17:44 .rsrc_SimpleText
-rw-r--r--   1 jhalter  staff       0 Jun  9 17:44 SimpleText
```

I'll have a solution for this eventually.

**Comment by jhalter** [comment]
This will be fixed in #14
